lib/store.js

```diff
@@ -82,8 +82,10 @@ class Store {
     this.current = null;
     this.state = new Meta();
     this.index = 0;
-    this.rootCache = new Map();
     this.lastMeta = new Meta();
+    // tracks root cache indexer pointer.
+    this.indexMeta = new Meta();
```

Comment:
Just so I'm clear: indexMeta is where we store the oldest Meta we have read from cache. If we look for a root and it's not in the cache and we set option cacheOnly to false, then this Meta is where we start looking for that missing root, and proceed backwards from there. So it's a pointer to where in our history we have stopped caching? If I'm correct I think we could figure out a better name for this member and maybe add a longer comment.

Reply:
yep. I agree, it's not the best name for it.
I called indexMeta, because process is indexing roots from the disk to the cache. (previously lastMeta was used for this job)
but it's more of a pointer to the last meta we have looked up from the disk in the history.